modules/37_industry/subsectors/datainput.gms

```diff
@@ -49,9 +57,144 @@ if (cm_IndCCSscen eq 1,
 *** assume 50 year lifetime for industry energy efficiency capital
 pm_delta_kap(regi,ppfKap_industry_dyn37) = -log(1 / 4) / 50;
 
-p37_energy_limit("ue_cement","en_cement")                     = 100;
-p37_energy_limit("ue_steel_primary","en_steel_primary")       = 100;
-p37_energy_limit("ue_steel_secondary","feel_steel_secondary") = 100;
+*** FIXME: this is temporary data, insert meaningful figures!
+p37_energy_limit("ue_cement")          =  10000;
+p37_energy_limit("ue_steel_primary")   =  10000;
+p37_energy_limit("ue_steel_secondary") = 100000;
+
+*' Emission factors for calculating industry emissions
+p37_fctEmi("fesos") = fm_dataemiglob("pecoal","sesofos", "coaltr","co2");
+p37_fctEmi("fehos") = fm_dataemiglob("peoil", "seliqfos","refliq","co2");
+p37_fctEmi("fegas") = fm_dataemiglob("pegas", "segafos", "gastr", "co2");
+
+*** CCS for industry is off by default
+emiMacSector(emiInd37_fuel) = NO;
+pm_macSwitch(emiInd37)      = NO;
+
+*** turn on CCS for industry emissions
+if (cm_IndCCSscen eq 1,
+  if (cm_CCS_cement eq 1,
+    emiMacSector("co2cement") = YES;
+    pm_macSwitch("co2cement") = YES;
+    pm_macSwitch("co2cement_process") = YES;
+    emiMac2mac("co2cement","co2cement") = YES;
+    emiMac2mac("co2cement_process","co2cement") = YES;
+  );
+
+  if (cm_CCS_chemicals eq 1,
+    emiMacSector("co2chemicals") = YES;
+    pm_macSwitch("co2chemicals") = YES;
+    emiMac2mac("co2chemicals","co2chemicals") = YES;
+  );
+
+  if (cm_CCS_steel eq 1,
+    emiMacSector("co2steel") = YES;
+    pm_macSwitch("co2steel") = YES;
+    emiMac2mac("co2steel","co2steel") = YES;
+  );
+);
+
+*** CCS for other industry is off in any case
+emiMacSector("co2otherInd") = NO;
+pm_macSwitch("co2otherInd") = NO;
+emiMac2mac("co2otherInd","co2otherInd") = NO;
+
+*** data on maximum secondary steel production
+Parameter 
+  p37_cesIO_up_steel_secondary(tall,all_regi,all_GDPscen)   "upper limit to secondary steel production based on scrap availability"
+  /
+$ondelim
+$include "./modules/37_industry/subsectors/input/p37_cesIO_up_steel_secondary.inc";
+$offdelim
+  /
+;
+
+s37_clinker_process_CO2 = 0.5262;
+
+*** FIXME *** this needs to be in moinput
```

Comment:
Could create compilation errors for non H12 regional set-up in case subsectors is activated.
Is it something you would like to correct before the merge or not?

Reply:
industry/subsectors won't work with anything other than H12 until input data is available.